nanoFirmwareFlasher/Program.cs

```diff
@@ -90,25 +90,33 @@ private static void CheckVersion()
         {
             Version latestVersion;
             Version currentVersion = Version.Parse(_informationalVersionAttribute.InformationalVersion.Split('+')[0]);
-
-            using (var client = new HttpClient())
+            try
```

Comment:
Let's have the PRs atomic and addressing one issue at the time.
Although relevant, this belong to another one.

Please remove these and make a separate PR.
As you are at it, there is no need to output any warning on failure to connect to github. This is not critical and not doing it is of no concern. So less noise for the console. 😉

Reply:
Removed from code until next pull request.